worlds/doom_1993/Options.py

```diff
@@ -0,0 +1,24 @@
+import typing
+
+from Options import Choice, Range, DeathLink, DefaultOnToggle, StartInventoryPool, Option
+
+
+# "I'm too young to die.", # Baby
+# "Hey, not too rough.", # Easy
+# "Hurt me plenty.", # Medium
+# "Ultra-Violence.", # Hard
+# "Nightmare!" # Nightmare
+class Difficulty(Choice):
+    """Choose the difficulty option. Those match DOOM's difficulty options."""
+    display_name = "Difficulty"
+    option_baby = 0
+    option_easy = 1
+    option_medium = 2
+    option_hard = 3
+    option_nightmare = 4
+    default = 2
+
+
+options: typing.Dict[str, type[Option]] = {
```

Comment:
This `type[Options]` is causing issue on some people's PC. A python version mismatch?
VSCode intellisense was telling me it was an error using parentheses `type(Option)`

Reply:
that would do it. `type(Option)` would too.